workers/generic-worker/artifacts.go

```diff
@@ -246,6 +246,11 @@ func (task *TaskRun) uploadLog(name, path string) *CommandExecutionError {
 }
 
 func (task *TaskRun) uploadArtifact(artifact artifacts.TaskArtifact) *CommandExecutionError {
+	if !task.Payload.Features.Artifacts {
+		task.Infof("Skipping upload of artifact %v as artifacts feature is not enabled", artifact.Base().Name)
```

Comment:
I guess maybe this log doesn't make sense since there'll be no live/backing logs anyway 😅